packages/client/src/comp/MockPage.tsx

```diff
@@ -16,10 +17,12 @@ const RawMockPage: React.FC<{}> = () => {
   const { state } = StateContainer.useContainer()
   const { setAddress } = AddressContainer.useContainer()
   const { setContact } = ContactContainer.useContainer()
+  const { setAddress: setAddressParts } = AddressInputPartContainer.useContainer()
```

Comment:
That would make editing individual inputs more repetitive since we'd need to always do a spread operation when updating `addressParts` in `<AddressInput/>`.

Reply:
Sorry -- I meant should `setAddress` be named `setAddressParts` since it takes a type `AddressInputParts`?  Agreed about `addressParts`.